Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM qazbnm456/fbctf_base:latest
```

Comment:
Is there a big benefit to having this base image? Can the official Ubuntu 14.04.x image be used instead?